search/reverse_geocoder.hpp

```diff
@@ -82,7 +82,8 @@ class ReverseGeocoder
   void GetNearbyStreets(FeatureType & ft, vector<Street> & streets) const;
   //@}
 
-  /// @returns [a lot of] nearby feature's streets and feature's street index, if valid ( < vector.size()).
+  /// @returns [a lot of] nearby feature's streets and an index of a feature's street.
+  /// If index is greater than vector.size() a feature belongs to no street.
```

Comment:
Returns a value greater than vector size when there are no Street the feature belongs to.